Server/VikingVault.API/Controllers/AutomaticPaymentController.cs

```diff
@@ -19,12 +21,27 @@ public AutomaticPaymentController(IAutomaticPaymentService automaticPaymentServi
         {
             _automaticPaymentService = automaticPaymentService;
         }
-        
+
         [HttpGet]
         public List<AutomaticPaymentDTO> GetAllAutomaticPayments()
         {
             var token = Request.Headers["x-access-token"];
             return _automaticPaymentService.GetAllAutomaticPayments(token);
         }
+
+        [HttpDelete]
+        [Authorization(Role = RoleEnum.User)]
+        public ActionResult Delete(AutomaticPaymentId automaticPaymentToDelete)
+        {
+            try
+            {
+                _automaticPaymentService.DeleteAutomaticPayment(automaticPaymentToDelete.Id);
+                return Ok(true);
```

Comment:
Why return `true` if it's never otherwise?

Reply:
An internal server error can also be returned in case of an exception from UserService.